js/supersized-karaoke.js

```diff
@@ -1,955 +1,983 @@
+          nextslide.css({left : -base.$el.width(), 'visibility': 'visible'}).animate({ left:0, avoidTransforms : false }, base.options.transition_speed, function(){ base.afterAnimation(); });
+          break;
+        case 4: case 'slideBottom': // Slide Bottom (reverse)
+          nextslide.css({top : -base.$el.height(), 'visibility': 'visible'}).animate({ top:0, avoidTransforms : false }, base.options.transition_speed, function(){ base.afterAnimation(); });
+          break;
+        case 5: case 'slideLeft':  // Slide Left (reverse)
+          nextslide.css({left : base.$el.width(), 'visibility': 'visible'}).animate({ left:0, avoidTransforms : false }, base.options.transition_speed, function(){ base.afterAnimation(); });
+          break;
+        case 6: case 'carouselRight': // Carousel Right (reverse)
+          nextslide.css({left : -base.$el.width(), 'visibility': 'visible'}).animate({ left:0, avoidTransforms : false }, base.options.transition_speed, function(){ base.afterAnimation(); });
+          liveslide.css({left : 0}).animate({ left: base.$el.width(), avoidTransforms : false}, base.options.transition_speed );
+          break;
+        case 7: case 'carouselLeft':   // Carousel Left (reverse)
+          nextslide.css({left : base.$el.width(), 'visibility': 'visible'}).animate({ left:0, avoidTransforms : false }, base.options.transition_speed, function(){ base.afterAnimation(); });
+          liveslide.css({left : 0}).animate({ left: -base.$el.width(), avoidTransforms : false }, base.options.transition_speed );
+          break;
+      }
+      return false;
+    };
+
+
+    /* Play/Pause Toggle
+    ----------------------------*/
+    base.playToggle = function(){
+      if (vars.in_animation || !api.options.slideshow) {
+        return false;   // Abort if currently animating
+      }
+
+      if (vars.is_paused){
+        console.log('starting slideshow');
+        vars.is_paused = false;
+
+        // Call theme function for play
+        if( typeof theme != 'undefined' && typeof theme.playToggle == "function" ) {
+          theme.playToggle('play');
+        }
+
+        // Resume slideshow
+        vars.slideshow_interval = setInterval(base.nextSlide, base.options.slide_interval);
+
+      } else{
+        console.log('stopping slideshow');
+        vars.is_paused = true;
+
+        // Call theme function for pause
+        if( typeof theme != 'undefined' && typeof theme.playToggle == "function" ) {
+          theme.playToggle('pause');
+        }
+        // Stop slideshow
+        clearInterval(vars.slideshow_interval);
+
+      }
+
+      return false;
+
+    };
+
+      /* Go to random guaranteed slide
+    ----------------------------*/
+    base.goToRandomGuaranteed = function() {
+      // base.goTo() starts counting from 1
+      slideIndex = Math.ceil(Math.random() * guaranteedSlides.length + vars.original_slide_length);
+      console.log('bozz slide number: ' + slideIndex);
```

Comment:
Change this log message, then we can contribute this back upstream.

Reply:
Hmm I don't think I added this...I think this log line is already upstream :joy:

Reply:
yeah it was from my accidental PR. but let's get rid of this upstream too.

Reply:
ok removed all bozz references